libs/scripts/delayed.js

```diff
@@ -12,12 +12,16 @@
 
 export const loadJarvisChat = async (getConfig, getMetadata, loadScript, loadStyle) => {
   const config = getConfig();
-  const jarvis = getMetadata('jarvis-chat');
-  if (!config.jarvis?.id || !config.jarvis?.version) return;
-  if (jarvis === 'on') {
-    const { initJarvisChat } = await import('../features/jarvis-chat.js');
-    initJarvisChat(config, loadScript, loadStyle);
-  }
+  const jarvis = getMetadata('jarvis-chat')?.toLowerCase();
+  if (!jarvis || !['mobile', 'desktop', 'on'].includes(jarvis)
+    || !config.jarvis?.id || !config.jarvis?.version) return;
+
+  const desktopViewport = window.matchMedia('(min-width: 900px)').matches;
+  if (jarvis === 'mobile' && desktopViewport) return;
+  if (jarvis === 'desktop' && !desktopViewport) return;
```

Comment:
Going deep into nitpicking territory, this could be written as a single check:
```
const jarvis = getMetadata('jarvis-chat')?.toLowerCase();
const desktopViewport = window.matchMedia('(min-width: 900px)').matches;

if (!jarvis
  || !['mobile', 'desktop', 'on'].includes(jarvis)
  || !config.jarvis?.id
  || !config.jarvis?.version
  || (jarvis === 'mobile' && desktopViewport)
  || (jarvis === 'desktop' && !desktopViewport)) return;
```

Reply:
That's correct, but it would require an additional (and useless) computation even if metadata is not defined.